devops/deployment/Deploy-Postgres-DB.sh

```diff
@@ -64,4 +60,15 @@ az postgres server firewall-rule create \
     --server-name $ServerName \
     --name "AllowMyIP_$RuleDate" \
     --start-ip-address $Local_IP_Address \
-    --end-ip-address $Local_IP_Address
\ No newline at end of file
+    --end-ip-address $Local_IP_Address
+
+echo
+echo "Create a firewall rule for Azure services"
+echo
+RuleDate=$(date +%F_%H-%M-%S)
+az postgres server firewall-rule create \
+    --resource-group $ResourceGroup \
+    --server-name $ServerName \
+    --name "AzureServices_$RuleDate" \
+    --start-ip-address "0.0.0.0" \
```

Comment:
@jmspring does 0.0.0.0 to 0.0.0.0 really work? I thought I tried that in the past and it didn't work. I've been using 0.0.0.0 to 255.255.255.255.

Reply:
Per - https://docs.microsoft.com/en-us/azure/postgresql/howto-manage-firewall-using-cli. it should be 0s.